zerver/openapi/zulip.yaml

```diff
@@ -4227,16 +4227,19 @@ paths:
         presence endpoint, which returns data for all active users in the
         organization, instead.
 
-        `GET {{ api_url }}/v1/users/{email}/presence`
+        `GET {{ api_url }}/v1/users/{user_id_or_email}/presence`
 
         See
         [Zulip's developer documentation](https://zulip.readthedocs.io/en/latest/subsystems/presence.html)
         for details on the data model for presence in Zulip.
       parameters:
-        - name: email
+        - name: user_id_or_email
           in: path
           description: |
-            The email address of the user whose presence you want to fetch.
+            The user_id or email address of the user whose presence you want to fetch.
+
+            **Changes**: New in Zulip 4.0 (feature level 39). Previous versions supproter only
+            user_id .
```

Comment:
Can you fix the typos here?  (`supported` mispelled, and an extra space before the `.` at the end of the Changes sentence).

For user-facing documentation like this, it's really important to have clear, copy-edited English, because typos really reduce confidence in the quality of documentation one's reading (I'm sure you've had this experience yourself).  So whenever editing a documentation file, please take special care to check for typos.

Reply:
Oh sorry I will take extra care next time.